### ISSUE TITLE
Project backup 404 not found?

### ISSUE THREAD

**Issue by lkthomas** [ansible-semaphore/semaphore#1812]
This is the Python script which intended to backup everything via the API, 
```
def get_endpoint_data(endpoint):
    """
    General function to fetch data from a given Semaphore API endpoint.
    """
    headers = {
        'Authorization': f'Bearer {API_TOKEN}'
    }
    try:
        response = requests.get(f'{SEMAPHORE_URL}/{endpoint}', headers=headers)
        response.raise_for_status()
        return response.json()
    except requests.exceptions.RequestException as err:
        print(f"Error retrieving data from endpoint {endpoint}: {err}")
        return None

def backup_project_data(project_id):
    """
    Backs up all data related to a given project, including tasks, history, etc.
    """
    # Hypothetical endpoints for tasks and history
    tasks_endpoint = f'projects/{project_id}/tasks'
    history_endpoint = f'projects/{project_id}/history'

    # Fetch tasks and history
    tasks = get_endpoint_data(tasks_endpoint)
    history = get_endpoint_data(history_endpoint)

    # Store or process this data as needed
    project_data = {
        'tasks': tasks,
        'history': history
    }
    # Example of saving the data as a JSON file
    with open(f'project_{project_id}_backup.json', 'w') as backup_file:
        json.dump(project_data, backup_file, indent=4)

    print(f"Backup completed for project {project_id}.")

def main():
    # Fetch all projects
    projects = get_endpoint_data('projects')

    if projects:
        for project in projects:
            project_id = project.get('id')
            # Trigger backup for each project
            backup_project_data(project_id)
    else:
        print("No projects to back up.")

if __name__ == "__main__":
    main()
```

I got this error:
Error retrieving data from endpoint projects/2/tasks: 404 Client Error: Not Found for url: http://localhost:3000/api/projects/2/tasks
Error retrieving data from endpoint projects/2/history: 404 Client Error: Not Found for url: http://localhost:3000/api/projects/2/history
Backup completed for project 2.
Error retrieving data from endpoint projects/3/tasks: 404 Client Error: Not Found for url: http://localhost:3000/api/projects/3/tasks
Error retrieving data from endpoint projects/3/history: 404 Client Error: Not Found for url: http://localhost:3000/api/projects/3/history
Backup completed for project 3.
Error retrieving data from endpoint projects/1/tasks: 404 Client Error: Not Found for url: http://localhost:3000/api/projects/1/tasks
Error retrieving data from endpoint projects/1/history: 404 Client Error: Not Found for url: http://localhost:3000/api/projects/1/history
Backup completed for project 1.

So I went to my browser to check out http://localhost:3000/api/projects/1/tasks, also showing 404 not found, what's wrong with this implementation?

**Comment by lkthomas** [comment]
I think I got a typo, it should be "project" and not "projects", and "history" parameter seems not exists

**Comment by lkthomas** [comment]
http://localhost:3000/api/project/1/backup <- this is documented on API reference but it shows 404 not found

**Comment by pierrediancourt** [comment]
Hello @lkthomas 

I was also wondering why this API endpoint was throwing a 404 with my v2.9.45 (latest stable) installation while being documented officially

Based on https://github.com/ansible-semaphore/semaphore/pull/1748  which shows that it this feature has been merged to the "develop" branch quite recently (https://github.com/ansible-semaphore/semaphore/commit/540669afd95b071e7878bb4076a285874bc58af8)
I can assume you're just like me :  not running the latest beta version containing this change.

I installed the version [v2.9.53-beta](https://github.com/ansible-semaphore/semaphore/releases/tag/v2.9.53-beta) and the api call works.

So this is a non issue, but I would advise devs/maintainers to set some versionning on the api doc page.
It's currently displaying v2.9.4 hence the legitimate confusion we encountered.

Thanks for the good work
